docs/project/SystemOverview.rst

```diff
@@ -18,20 +17,21 @@ Overview of system components:
 
 The cloud and the web application are developed using `TypeScript <https://www.typescriptlang.org/>`_ (a typed superset of JavaScript).
 JavaScript is the most popular language according to the `2019 Stack Overflow survey <https://insights.stackoverflow.com/survey/2019#technology>`_.
-Most of the cloud providers provide their SDKs in JavaScript.
+Most of the cloud providers provide the SDKs in JavaScript.
```

Comment:
```suggestion
Most of the cloud providers provide their SDKs in JavaScript.
```
SDKs are never mentioned before so it's not clear which are referred to here if using `the`.